ksql-engine/src/test/java/io/confluent/ksql/util/StringUtilTest.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2017 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.util;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+/**
+ * Unit tests for class {@link StringUtil}.
+ *
+ * @see StringUtil
+ */
+public class StringUtilTest {
+
+  @Test
+  public void testCleanQuotesWithNonEmptyString() {
```

Comment:
For these tests we probably should have `@Test(expected=KsqlException.class)`
Also, I think the naming could be more descriptive, i.e., what should it do?
something like `shouldThrowKsqlExceptionWhenQuotesNotClosed`